vcd/resource_vcd_nsxt_ip_set_test.go

```diff
@@ -170,6 +170,7 @@ func TestAccVcdNsxtIpSet(t *testing.T) {
 					resource.TestCheckTypeSetElemAttr("vcd_nsxt_ip_set.set1", "ip_addresses.*", "2001:db8::/48"),
 					resource.TestCheckTypeSetElemAttr("vcd_nsxt_ip_set.set1", "ip_addresses.*", "2001:db6:0:0:0:0:0:0-2001:db6:0:ffff:ffff:ffff:ffff:ffff"),
 					resource.TestCheckResourceAttr("vcd_nsxt_ip_set.set1", "ip_addresses.#", "5"),
+					resource.TestMatchResourceAttr("vcd_nsxt_ip_set.set1", "owner_id", regexp.MustCompile(`^urn:vcloud:vdc:.*$`)),
```

Comment:
Looks like you have `owner_id` field in the state of the vcd_nsxt_ip_set. What about adding it to the documentation as a read-only attribute?

Reply:
For routed networks I did not add it to docs (but still put the field itself) because it might not be correct after update if edge gateway was moved from VDC to VDC Group.

The situation could go as follows if only `owner_id` is changed for Edge Gateway resource during an update:
* routed network is refreshed (and owner_id is set to VDC, because Edge Gateway has not been moved yet)
* Edge Gateway owner_id is changed from VDC to VDC Group 

Only after second refresh/apply `owner_id` would reflect correct name. It is much more reliable to reference Edge Gateway `owner_id` than the one for a child.

This is true for those objects which migrate automatically with parent edge gateway.